### PR TITLE
layout Completed

### PR DESCRIPTION
**Pull request by sameerddr**

### CODE REVIEW

File: src/Components/Layout/StockLayout.css [hunk 1]
```diff
@@ -0,0 +1,15 @@
+.layoutHeader {
+    background-color: aqua;
+    height: 12vh;
+}
+
+#ant-layout-sider {
+    background-color: rgb(197, 197, 30);
+    min-width: 400px !important;
```
Comment (Souvikdeb2612) [inline src/Components/Layout/StockLayout.css]: Avoid using important. Use component customization or css chaining

File: src/App.js [hunk 2]
```diff
@@ -1,6 +1,6 @@
 import "antd/dist/antd.css";
 import "./App.css";
```
Comment (Souvikdeb2612) [inline src/App.js]: CSS imports should occur in the end of all imports and add a space after imports and before starting the function

File: src/Components/Layout/StockLayout.jsx [hunk 3]
```diff
@@ -1,17 +1,17 @@
 import { Layout } from "antd";
 import React from "react";
+import "../Layout/StockLayout.css";
```
Comment (Souvikdeb2612) [inline src/Components/Layout/StockLayout.jsx:4]: Add a space after the imports. Also fix the relative path, edit out the unnecessary path.

File: src/Components/Layout/StockLayout.jsx [hunk 4]
```diff
@@ -1,17 +1,17 @@
 import { Layout } from "antd";
 import React from "react";
```
Comment (Souvikdeb2612) [inline src/Components/Layout/StockLayout.jsx:1]: React import should be at the top along with all other internal imports, then a space and all external imports, another space and finally css imports

**Review by Souvikdeb2612 — CHANGES_REQUESTED** [review]

**Comment by sameerddr** [comment]
Sir can u use important as without this it could be hectic... We are using
important only in 1 line

File: src/Components/Layout/StockLayout.css (hunk 1 above)
Comment (sameerddr) [reply src/Components/Layout/StockLayout.css]: Issue is Solved Plx Go throug It

**Review by Souvikdeb2612 — APPROVED** [review]

**Pull request merged**